File: src/test/samples/sample.erl
%% #region FirstRegion
X = 42.
%% #endregion

%% #region SecondRegion
-module(my_class).
-export([my_method/0]).

%% #region InnerRegion
my_method() -> ok.
%% #endregion

% #region
method_in_unnamed_region() -> ok.
%%%   #endregion

%% #endregion

File: src/test/samples/sample.m
//#region FirstRegion
#import <Foundation/Foundation.h>
//#endregion

//  #region SecondRegion
@interface MyClass : NSObject
//  #region    InnerRegion  
- (void)myMethod;
//  #endregion  ends InnerRegion   

//  #region
@property (nonatomic, strong) NSString *name;
// #endregion
@end
// #endregion
